models/Dish.js

```diff
@@ -6,7 +6,7 @@ const DishSchema = new Schema(
     name: { type: Schema.Types.ObjectId, ref: "TextString" },
     description: { type: Schema.Types.ObjectId, ref: "TextString" },
     businessId: { type: Schema.Types.ObjectId, ref: "Business" },
-    image: { data: Buffer, contentType: String },
+    image: {},
```

Comment:
Should be the S3 key string, no? relevant to other models too

Reply:
It's two keys so it's an object. It doesn't have to be in the schema

Reply:
I can create an image schema and have it as an embedded document, what do you think?